miio/gateway.py

```diff
@@ -647,69 +647,79 @@ class GatewayLight(GatewayDevice):
 
     @command()
     def rgb_status(self):
-        """Get current status of the light (always correct)."""
+        """
+        Get current status of the light.
+        Always represents the current status of the LEDs as oposed to while night_light_status.
+
+        Example:
+           {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)}
+        """
         # Returns {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)} when light is off
         state_int = self._gateway.send("get_rgb").pop()
         brightness = int_to_brightness(state_int)
         rgb = int_to_rgb(state_int)
-        if brightness > 0:
-            is_on = True
-        else:
-            is_on = False
+        is_on = brightness > 0
+
         return {"is_on": is_on, "brightness": brightness, "rgb": rgb}
 
     @command()
     def night_light_status(self):
-        """Get status of the night light."""
-        # The night light controls the same LEDs as the rgb light, but has a seprate memory for brightness and color.
-        # Changing the rgb light does not effect the stored state of the night light, while changing the night light does effect the state of the rgb light.
-        # Therefore this command only gives the correct status of the LEDs if the last command was a night light command and not a rgb light command, otherwise it gives the stored values of the night light.
-        # Returns {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)} when night light is off
+        """
+        Get status of the night light.
+
+        The night light controls the same LEDs as the rgb light, but has a seprate memory for brightness and color.
+        Changing the rgb light does not effect the stored state of the night light, while changing the night light does effect the state of the rgb light.
+        Therefore this command only gives the correct status of the LEDs if the last command was a night light command and not a rgb light command, otherwise it gives the stored values of the night light.
```

Comment:
Some sort of explanation to the class' docstring would be great, I think this could (at least partially) belong there as it description itself is not just about the night light but the whole light support.

What do you think?

Reply:
You are right!
I will change it, just a moment

Reply:
I made a more clear explanation in the class string, and removed the duplicate info here.